notebooks/GP-Marginal.ipynb

```diff
@@ -0,0 +1,381 @@
+{
```

Comment:
<ul><li>A Marginal Gaussian process jointly represent<s>ing</s><strong>s</strong> the data as a large probability distribution</li><li>making it easy to infer a distr<strong>b</strong>ution over the test data for prediction and generation.</li><li>Maybe explain why adding noise is necessary and useful</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc4/pull/309/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>